docs/milestone2.md

```diff
@@ -81,12 +79,12 @@ import pyad
 
 Specific classes or functions can be called  individual and will operate independently, for example:
 ```python
-from pyad import forwardmode
+from pyad import forward_diff
 ```
 
 ### Interaction Theory
 
-In general the **pyad** package will work on an object oriented, class based approach similar to sklearn or other similar modules. **pyad** will contain a number of classes which can be instantiated - these will be classes such as `forwardmode`. The user will create a blank instance of the differentiator object which will then persist. By design this will be a blank slate and there will not be a specific set of default inputs as each user may have a very different use case (differentiating a single variable or multi-variable function for instance).
+In general the **pyad** package will work on an object oriented, class based approach similar to sklearn or other similar modules. **pyad** will contain a number of classes which can be instantiated - these will be classes such as `MultivariateDerivative`. The user will create functions to be differentiated and initialize variables. There will not be a specific set of default inputs as each user may have a very different use case (differentiating a single variable or multi-variable function for instance).
```

Comment:
So, I think that the users shouldn't really ever need to instantiate `MultivariateDerivative`, but they will have to instantiate `Variable` and may occassionally want to instiante `Tensor`.